server/boot/root.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(server) {
```

Comment:
Not sure its possible to make that detection. In that case, I'll just leave it here and if needed the parent app can override it.